src/components/generate_prompts/src/main.py

```diff
@@ -112,8 +112,11 @@ def load(self) -> dd.DataFrame:
         room_tuples = itertools.product(rooms, interior_prefix, interior_styles)
         prompts = map(lambda x: make_interior_prompt(*x), room_tuples)
 
-        pandas_df = pd.DataFrame(prompts, columns=["prompts_text"])
+        pandas_df = pd.DataFrame(prompts, columns=["prompt"])
+        pandas_df['id'] = range(1, len(pandas_df) + 1)
```

Comment:
A `RangeIndex` should be created automatically ([docs](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.html)).

I assume this "id" field might just be filtered out while writing since we don't set it as index and it's not defined in the produces section?